src/main/java/games/strategy/triplea/ui/menubar/HelpMenu.java

```diff
@@ -259,12 +260,9 @@ private void addGameNotesMenu(final JMenu parentMenu) {
         if (dialog.getHeight() > 600) {
           dialog.setSize(dialog.getWidth(), 600);
         }
-        // dialog.setLocationRelativeTo(frame);
         dialog.addWindowListener(new WindowAdapter() {
           @Override
           public void windowOpened(final WindowEvent e) {
-            scroll.getVerticalScrollBar().getModel().setValue(0);
-            scroll.getHorizontalScrollBar().getModel().setValue(0);
```

Comment:
This slightly changes the behavior of the game notes panel.
Previously it would scroll back to the start whenever the panel is reopened.
Now it just starts where you last left off. Better IMO